services/worker/README.md

```diff
@@ -102,13 +102,15 @@ Set environment variables to configure the `duckdb-index` worker (`DUCKDB_INDEX_
 Set environment variables to configure the `descriptive-statistics` worker (`DESCRIPTIVE_STATISTICS_` prefix):
 
 - `DESCRIPTIVE_STATISTICS_CACHE_DIRECTORY`: directory to which a dataset in parquet format is downloaded. Defaults to empty.
+- `DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES`: size in bytes by which the number of first `n` parquet files to compute statistics over is determined, satisfies the following condition: `sum(parquet_files[0] + ... + parquet_files[n-1]) <= DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES & sum(parquet_files[0] + ... + parquet_files[n]) > DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES`. Defaults to `100_000_000`.
 - `DESCRIPTIVE_STATISTICS_HISTOGRAM_NUM_BINS`: number of histogram bins (see examples below for more info).
 - `DESCRIPTIVE_STATISTICS_MAX_PARQUET_SIZE_BYTES`: maximum size in bytes of the dataset's parquet files to compute statistics. Datasets with bigger size are ignored. Defaults to `100_000_000`.
-
+variable
 #### How descriptive statistics are computed 
 
 Descriptive statistics are currently computed for the following data types: strings, floats, and ints (including `ClassLabel` int). 
-Response has two fields: `num_examples` and `statistics`. `statistics` field is a list of dicts with three keys: `column_name`, `column_type`, and `column_statistics`.
+Response has three fields: `num_examples`, `statistics` and `partial`. `partial` indicates if statistics are computed over first ~`DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES` of a dataset. If it equals to `False`, that means that `num_examples` correspond to the number of examples in this first chunk of data, not of the full dataset. 
```

Comment:
wow, it's complex. Is it possible to change this, so that, if `partial: False`, we are sure that the statistics are for the full split? Else, it's not useful for the Hub, it does not bring enough information.

Reply:
> If it equals to `False`, that means that `num_examples` correspond to the number of examples in this first chunk of data, not of the full dataset. 

hmmm, from the code, I understand that if it's False, then all the split's data have been used, no?